### PR TITLE
merge in stats branch

### PR DESCRIPTION
**Pull request by DrewDennison**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -86,8 +89,8 @@ workflows:
   weekly-stats:
     triggers:
       - schedule:
-          # Run at 10:00 every Sunday, UTC.
-          cron: "0 10 * * 3"
+          # Run at 00:00 every day, UTC.
+          cron: "0 0 * * *"
```
Comment (mjambon) [inline .circleci/config.yml:93]: You should rename the job to `daily-stats`.

File: scripts/upload-stat.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
+#! /usr/bin/env python3
+#
+# Parse the stat.txt file from lang-stat and upload to the metrics collection endpoint
```
Comment (mjambon) [inline scripts/upload-stat.py:3]: Thank you!

(we could have made the shell script produce json, instead of parsing the `txt` file here)

Comment (DrewDennison) [reply scripts/upload-stat.py:3]: yeah I thought about it but figured it wasn't that hard to parse your format and I like your human readable format
